Readme.md

```diff
@@ -4,19 +4,72 @@
 [![NPM Downloads][downloads-image]][downloads-url]
 [![Build Status][travis-image]][travis-url]
 
-Utility to parse a string bytes (ex: `1TB`) to bytes (`1099511627776`) and vice-versa.
+Utility to parse a string bytes (ex: `1TB`) to bytes (`1,000,000,000,000`) and vice-versa.
+
+This uses the byte units defined in ISO/IEC 80000-13:2008, both the binary prefixes and the original SI units.
+
+Supported units and abbreviations are as follows and are case-insensitive:
+
+### Metric/Decimal Prefixes
```

Comment:
You shouldn't be using the heading level for the style/font size, because different sites render differently. Only h2 should be under h1, rather than skipping to h3.